position_controller:
  ros__parameters:
    limit:
      v: 1.0
      w: 1.0

    controller_gain:
      k1: 0.2
      k2: 1.2
      dTol: 0.1

    topic:
      pub:
        twist: /cmd_vel
      sub:
        odometry: /odom
        goal: /goal_pose
